services/compute/virtual-machines/taxonomy.md

```diff
@@ -0,0 +1,47 @@
+# Virtual Machines Taxonomy
+
+This _service-level taxonomy_ documents the minimal set of features
+that should be present for a service to be considered portable for
+use in financial services ecosystems.
+
+## Taxonomy
+
+| Taxonomy ID | Feature | Description |
+| ----------- | ------- | ----------- |
+|	CCC-030101	|	Instance Types	|	Offers a range of instance types with different specifications for CPU, GPU, memory, storage optimisation and networking capacity.	|
+|	CCC-030102	|	Operating System Options 	|	A selection of operating systems for virtual machine instances.	|
+|	CCC-030103	|	Ephemeral Storage	|	Temporary storage available to the VM which is lost when the instance is stopped or terminated	|
+|	CCC-030104	|	Availability	|	Ensuring high availability of virtual machine instances through redundancy and multiple (availability) zones within a region.	|
+|	CCC-030105	|	Identity and Access Management	|	Implementing identity and access management features such as key pairs, JIT and MFA to control user access to virtual machine instances.	|
+|	CCC-030106	|	Monitoring and Logging	|	Offering monitoring and logging capabilities to track performance metrics, user access and security events.	|
+|	CCC-030107	|	Backup and Restore	|	Providing backup and disaster recovery solutions for virtual machine instances and associated data, including snapshot-based backups, incremental backups, and point-in-time recovery.	|
+|	CCC-030108	|	Encryption at Rest	|	Encrypting data stored by virtual machine instances to protect against unauthorized access.	|
```

Comment:
Isn't this a feature on EBS? VM doesn't have a storage of it's own.

Reply:
I was thinking the ephemeral storage can be encrypted? I'll double check though, I imagine it needs integrating with KMS or similar.